srcpkgs/pass-update/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'pass-update'
+pkgname=pass-update
+version=2.1
+revision=1
+archs=noarch
+build_style=gnu-makefile
+depends="pass"
+short_desc="A pass extension providing an easy flow for updating passwords"
```

Comment:
Please remove the article, according to our convention descriptions should not start with one. Just `Pass extension providing an easy flow for updating passwords` is fine.

Reply:
Changed this